torch/cuda/tunable.py

```diff
@@ -102,14 +107,70 @@
 given set of input arguments (transa, transb, m, n, k) will attempt to use the
 fastest available implementation across both rocblas and hipblaslt.
 
+Offline Tuning
+==============
+
+Motivation
+----------
+There are several use cases for offline tuning.
+
+One use case involves a workload with a high-memory utilization, where regular tuning might lead to running out of memory.
+
+Another use case is for compute-intensive workloads. In such cases, it is more resource-efficient to collect
+the GEMMs for the workload once and then tune repeatedly with different tuning parameters or libraries.
+
+Workflow
+--------
+There are basically two steps:
+1) Set the environment variables to collect the untuned GEMM and this will generate ``tunableop_untuned0.csv``::
+
+  PYTORCH_TUNABLEOP_ENABLED=1
+  PYTORCH_TUNABLEOP_TUNING=0
+  PYTORCH_TUNABLEOP_RECORD_UNTUNED=1
+  ...
+
+2) Run a Python script that reads the `tunableop_untuned0.csv` and generates the `tunableop_results0.csv`, like::
```

Comment:
I'm not sure what's causing the doc build error but I would try:

* using `.. code-block:: python` directive instead of `::`
* make sure indentation is correct, like:

```
.. code-block:: python

   PYTORCH_TUNABLEOP_ENABLED=1
   PYTORCH_TUNABLEOP_TUNING=0
   PYTORCH_TUNABLEOP_RECORD_UNTUNED=1
   ...
```